lms/envs/aws.py

```diff
@@ -272,6 +272,12 @@
 # Queue to use for updating persistent grades
 RECALCULATE_GRADES_ROUTING_KEY = ENV_TOKENS.get('RECALCULATE_GRADES_ROUTING_KEY', LOW_PRIORITY_QUEUE)
 
+# Queue to use for updating persistent social engagements
+RECALCULATE_SOCIAL_ENGAGEMENT_ROUTING_KEY = ENV_TOKENS.get(
+    'RECALCULATE_SOCIAL_ENGAGEMENT_ROUTING_KEY',
+    LOW_PRIORITY_QUEUE
```

Comment:
Since `aws.py` imports all of `common.py`, `RECALCULATE_SOCIAL_ENGAGEMENT_ROUTING_KEY` is actually already defined and available here, so we should default to that. This way we don't have to change this `LOW_PRIORITY_QUEUE` in 2 places if we want to change the default for all environments.

Reply:
And yeah, the other vars are doing it wrong, but going a bit higher they are doing it right, e.g. `BULK_EMAIL_DEFAULT_RETRY_DELAY`.

Reply:
I think we can ignore this comment, since it seems `LOW_PRIORITY_QUEUE` in aws.py is different than in common.py